### PR TITLE
Add basic attempt at an authentication doc; ref #5

### PR DESCRIPTION
**Pull request by greggilbert**
Just a quick doc based on my rudimentary setup of Warden. Almost certainly room for improvement, but it's a start.

### CODE REVIEW

**Comment by davydovanton** [comment]
Thanks for contribution 👍 
This PR looks good for me. But I think we need to change `sequel` to `hanami-model` in this example.

What do you think @cllns?

File: authentication/warden.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+  or in an application:
+
+  ```ruby
+  module Web
+    class Application < Hanami::Application
+      configure do
+        middleware.use Rack::Session::Cookie, secret: ENV['COOKIE_SECRET']
+
+        middlware.use Warden::Manager do |manager|
```
Comment (jasoncharnes) [inline authentication/warden.md]: I think there's a typo here. `middleware` instead of `middlware` :)

Comment (greggilbert) [inline authentication/warden.md]: Fixed.

**Comment by AlfonsoUceda** [comment]
as @davydovanton said, we should use Hanami::Model instead Sequel. @greggilbert thans for the PR but could you update with hanami-model?

**Comment by AlfonsoUceda** [comment]
ping @greggilbert

**Comment by cllns** [comment]
@AlfonsoUceda, @davydovanton and I are working on building a more comprehensive auth solution for Hanami, so I'm not sure this is worth updating. We'll still use the code from this PR for reference though.

**Comment by greggilbert** [comment]
@AlfonsoUceda Ah sorry, I got really busy with a bunch of other stuff and didn't get the chance to write more. I'll defer to @cllns' work.

**Comment by cllns** [comment]
Gonna close this for now then. Thanks for the work @greggilbert, I'll definitely reference the code here in the future

**Pull request closed without merging**